src/client/src/components/FileButton/styles.scss

```diff
@@ -2,7 +2,7 @@
     padding: 8px 14px;
     color: #fff;
     height: 30px;
-    font-family: monospace;
+    font-weight: bold;
```

Comment:
It was also possible to add cursor: pointer when hovering over the button

Reply:
Agree. For now, we created the MVP of the product, so those changes could be done in the next versions of the product.